pkgs/applications/office/watson/default.nix

```
@@ -13,6 +13,10 @@ buildPythonApplication rec {
     sha256 = "sha256-/AASYeMkt18KPJljAjNPRYOpg/T5xuM10LJq4LrFD0g=";
   };
 
+  patches = [
+    ./fix-completions.patch # https://github.com/TailorDev/Watson/pull/473
```

Can we fetchpatch this?

---

I'm not sure what that is, but I will investigate.

In related news, turns out there's some other issues with completion since they updated the click dependency, so I'll mark this WIP until I can look into it.

---

```suggestion
    # https://github.com/TailorDev/Watson/pull/473
    (fetchpatch {
      name = "fix-completion.patch";
      url = "https://github.com/TailorDev/Watson/commit/8ee6ef0f307fce6a8a93018652449604cd9937cf.patch";
      sha256 = "sha256-/vqY4eMzvealw2BdoG5jGzLMNGqZ3Oq5K+uGTH0Fyaw=";
    })
```

And include `fetchpatch` above.

---

I'll integrate this suggestion once I fixed the actual completion, thank you for spelling it out, that helps a lot :)